source/blog/2017-02-27-2017-ember-community-survey.html.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "2017 Ember Community Survey"
```

Comment:
There's no need to rename the 2017 blog post file.

Reply:
My bad, was changing it in a different file. Accidentally made the change in that file.